src/helpers/index.ts

```diff
@@ -21,3 +22,11 @@ export const getRegistrarForDomain = async (
   const contract = getRegistrar(hub.provider, registrarAddress);
   return contract;
 };
+
+export const sortDomains = (domains: Domain[]): Domain[] => {
+  return domains.sort((a, b) => {
+    if (a.id < b.id) return -1;
```

Comment:
Might be safer to turn the id into a `BigNumber` and then use the equality operators vs the string equality operators